css/main.css

```diff
@@ -488,3 +488,128 @@ footer {
     padding: 0px 250px 80px 210px;
   }
 }
+
+/***********************/
+/* EMAIL CAPTURE FORM */
+/***********************/
+
+.section-cta {
+  padding: 4.8rem 15.4rem;
+}
+
+.cta {
+  display: grid;
+  border-radius: 9px;
+  align-items: center;
+  justify-content: center;
+
+  box-shadow: 0 1.2rem 2.4rem rgba(0, 0, 0, 0.1);
+}
+
+.cta-text-box {
+  padding: 3.6rem 4.8rem 4.8rem;
+  color: #041484;
+}
+
+.cta .heading-secondary {
+  font-size: 2.4rem;
+  margin-bottom: 1.8rem;
+  color: inherit;
+}
+
+.cta-text {
+  font-size: 1.2rem;
+  line-height: 1.8;
+  margin-bottom: 2.4rem;
+}
+
+.cta-form {
+  display: grid;
+  grid-template-columns: 60% 30%;
+  column-gap: 3.6rem;
+  row-gap: 1.2rem;
+}
+
+.cta-form label {
+  display: block;
+  font-size: 1.2rem;
+  font-weight: 500;
+  margin-bottom: 1.2rem;
+}
+
+.cta-form input {
+  width: 100%;
+  padding: 1.2rem;
+  font-size: 1.2rem;
+  font-family: inherit;
+  border: none;
+  border-radius: 9px;
+
+  color: #777;
+  background-color: #edf2ff;
+  box-shadow: 0 1px 2px rgba(0, 0, 0, 0.1);
+}
+
+.cta-form input:focus {
+  outline: 3px solid #db09795d;
+}
+
+.btn-cta,
+.btn-cta:link,
+.btn-cta:visited {
```

Comment:
why are we using :link and :visited selector here?

Reply:
I have included them to keep the design consistent in case someone changes the code in future. It's a good practice to always include link,  visited, active, hover all at the same place(In case some component specific changes are to be made). I can remove it if want.

Reply:
I am not sure if there is a :link and :visited selector for button. They are used for links I guess

Reply:
I got confused with links and buttons. Indeed there are no use of visited and link pseudo classes here. Thanks for pointing that out.